### ISSUE TITLE
Build failure on openSUSE 15.4

### ISSUE THREAD

**Issue by bubbleguuum** [cyring/CoreFreq#419]
Building CoreFreq for openSUSE 15.4 on OBS (Open Build Service) fails on a ` label at end of compound statement` warning turned into error due to `-Wfatal-errors` being used:

```
[   45s]   CC [M]  /home/abuild/rpmbuild/BUILD/CoreFreq-1.95.4/corefreqk.o
[   46s] /home/abuild/rpmbuild/BUILD/CoreFreq-1.95.4/corefreqk.c: In function 'Query_TGL_IMC':
[   46s] /home/abuild/rpmbuild/BUILD/CoreFreq-1.95.4/corefreqk.c:5197:1: error: label at end of compound statement
[   46s]  EXIT_TGL_IMC:
[   46s] compilation terminated due to -Wfatal-errors.
```

On openSUSE Tumbleweed there is no such issue. Tumbleweed uses gcc 12 while I think 15.4 uses gcc 7 to compile modules

Googling this warning it seems to be caused by a `switch` without a `default` label or a `default` label without a `break`. 

[Full build log](https://build.opensuse.org/build/hardware/SLE_15_SP4/x86_64/CoreFreq/_log)

**Comment by bubbleguuum** [comment]
Thanks, confirming it fixes the compile error.

**Comment by olejon** [comment]
Confirmed Ubuntu 20.04.5 LTS. Stopped compiling with exit 1 error for `dkms install corefreqk/1.95` - not `dkms add corefreqk/1.95` first of course, but 1 second into `make` - on v 1.95.3 and also 1.95.4, so I reverted to 1.95.2 (was stable and reliable as always), as I assumed this was reported and so I didn't report it. Didn't bother you check the dkms error log, so.

Sorry. I should have when 2 consecutive releases failed.

Now it's fixed 👍🏻
